FormSubmissionAndValidation2/src/components/CustomButton.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+export const CustomButton = React.memo(function CustomButton(props) {
+  const buttonText = props.text;
+  const buttonAction = props.action;
+  return <>
+    <div className="centerContent">
+      <button id="buttonGo" className="button" onClick={buttonAction}>{buttonText}</button>
```

Comment:
ButtonText and action can be nil here. What should happen in that case?

Reply:
made the fields required.